README.md

```diff
@@ -226,8 +226,66 @@ within it. This allows for any unexpected state data of the user from hanging ar
 # Deploying the app
 ## If you decide to deploy to Heroku:
 * [Heroku Command Line Interface](https://devcenter.heroku.com/categories/command-line)
+* If you have HTTPS enabled (requires a paid dyno), be sure to enable `config.force_ssl = true` on line 47 of `config/environments/production.rb`
+
 ## If you decide to use Docker:
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) to use the docker-compose steps listed above
+
+_Note_: These instructions assume you are working with shell access to a system with Docker and docker-compose available (such as a Digital Ocean Docker image).
+
+_Note_: These instructions assume you already have a domain name registered and pointed it at the
+IP address of your server.
+
+1. You'll need to get the code on the server. The recommended way to do this is with a [GitHub Deploy Key](https://developer.github.com/v3/guides/managing-deploy-keys/). Deploy Keys are just SSH keys that are tied to a project, instead of being tied to a user, which means that they can persist independently of the project's membership. You can use [GitHub's guide on generating SSH keys to generate your deploy key](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+2. After you've generated your deploy key and added it to your project, clone the project.
+3. Create a Caddyfile. It can live anywhere (Mutual Aid Tompkins has ours at `~/.config/Caddyfile`), and it should look like this, replacing `your.domain.name` with your organization's desired domain name:
```

Comment:
I'm going to try and run through these now and let you know if I have any questions

Reply:
The instructions are pretty okay so far. I ran into some problems that are related to my environment because they're happening with other projects, too, now, so I'm going to wipe my setup and try again tomorrow.

As I'm going through this, I think we might need to split up the documentation so it's clearer what steps are for development environments and which are for production.

As we're expecting new people volunteering later this year, I think we'd be best off with development environment guidelines very streamlined and front and center (I imagine people on Windows 10 will get a lot of use out of setting up a Docker development environment). Then instructions that are specific for production could be in a separate file, maybe, with a link?

Definitely open to other ways of organizing this. Let me know what you think

Reply:
I was thinking same thing about the README -- finding parts that can be extracted to other files to make the outline really clear and everything else be links.

Reply:
100% agreed with this! Let me know if you still can't get things working with a fresh environment; I'm _pretty_ sure this is all of the steps I took on a fresh Digital Ocean droplet, but I absolutely might have missed something.